src/async-components/views/dialogs/secretstorage/CreateSecretStorageDialog.js

```diff
@@ -268,6 +277,22 @@ export default class CreateSecretStorageDialog extends React.PureComponent {
         return this.state.zxcvbnResult && this.state.zxcvbnResult.score >= PASSWORD_MIN_SCORE;
     }
 
+    _renderPhaseRestoreKeyBackup() {
+        const DialogButtons = sdk.getComponent('views.elements.DialogButtons');
+        return <div>
+            <p>{_t(
+                "Key Backup is enabled on your account but has not been set " +
+                "up from this sign-in. To set up secret storage, " +
```

Comment:
We changed to calling them "session" instead of "sign-in". The designs haven't been completely updated to match.